src/styles/blocks/card.css

```diff
@@ -0,0 +1,111 @@
+body {
+  background-color: var(--midium-bg-color);
+}
+
+.card {
+  min-width: 200px;
+  display: flex;
+  padding: calc(2 * var(--card-standard-padding))
```

Comment:
No need to declare a variable for value that is used once, this would be more readable if you would just do
```
padding: 32px 16px 16px;